misk/src/main/kotlin/misk/clustering/Cluster.kt

```diff
@@ -23,23 +23,27 @@ interface Cluster {
     /** @property Member The member representing this instance of the service */
     val self: Member,
 
-    /** @property Boolean true if the current service instance is ready as perceived by the cluster manager */
-    val selfReady: Boolean,
-
     /** @property Set<Member> of the ready members of the cluster */
     val readyMembers: Set<Cluster.Member>,
 
-    /** @property ClusterHashRing built from the ready members of this cluster */
-    val hashRing: ClusterHashRing = ClusterHashRing(readyMembers)
-  ) {
+    /** @property Boolean true if the current service instance is ready as perceived by the cluster manager */
```

Comment:
is the `@property Boolean` part useful? Presumably people reading the source code don’t need that. If/when we use Dokka (Kotlin’s Javadoc) I don’t think it’ll be useful either?

Reply:
you mean is the `@property` tag useful? just following the KDoc convention for it (and it bitches if the `Boolean` isn't specified. i don't really care though so happy to remove the tags

Reply:
I think we’re better off without the tags. I’ll remove ’em as I go.